.gitignore

```diff
@@ -48,3 +48,5 @@ gradle-app.setting
 .idea/
 *.iml
 out/
+/bin/
+
```

Comment:
I didn't get why it was added, but when I'm removing it the build is getting failed.Any suggestions how to fix that?